network-map-viewer/.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "tabWidth": 4,
```

Comment:
you shouldn't use tabWith : 4 with the linewith 80. 
so many line goes to 2 lines or more